docs/tutorials/data_management.md

```diff
@@ -0,0 +1,243 @@
+
+# 4. Data slicing
+
+You might have noticed that the blob we generated is a hefty 13MB. This is no surprise, as we included `--all-keys` `--all-locales`. However, our binary only uses date formatting data in Japanese. There's room for optimization:
+
+```console
+$ icu4x-datagen --cldr-tag latest --icuexport-tag latest --out my-data --format blob --keys-for-bin target/debug/myapp --locales ja
+```
+
+The `--keys-for-bin` argument tells `icu4x-datagen` to analyze the binary and only include keys that are used by its code. In addition, we know that we only need data for the Japanese locale. This significantly reduces the blob's file size, to 54KB, and our program still works. Quite the improvement!
+
+But there is more to optimize. You might have noticed this in the output of the `icu4x-datagen` invocation, which lists 21 keys, including clearly irrelevant ones like `datetime/ethopic/datesymbols@1`. Remember how we had to convert our `DateTime<Gregorian>` into a `DateTime<AnyCalendar>` in order to use the `DateTimeFormatter`? Turns out, as `DateTimeFormatter` contains logic for many different calendars, datagen includes data for all of these as well.
+
+We can instead use `TypedDateTimeFormatter<Gregorian>`, which only supports formatting `DateTime<Gregorian>`s:
+
+```rust
+use icu::locid::{locale, Locale};
+use icu::calendar::{DateTime, Gregorian};
+use icu::datetime::{TypedDateTimeFormatter, options::length};
+use icu_provider_blob::BlobDataProvider;
+
+const LOCALE: Locale = locale!("ja");
+
+fn main() {
+    let blob = std::fs::read("my-data").expect("Failed to read file");
+    let buffer_provider = 
+        BlobDataProvider::try_new_from_blob(blob.into_boxed_slice())
+            .expect("Failed to initialize Data Provider.");
+
+    let options = length::Bag::from_date_time_style(length::Date::Long, length::Time::Medium);
+
+    let dtf = TypedDateTimeFormatter::<Gregorian>::try_new_with_buffer_provider(&buffer_provider, &LOCALE.into(), options.into())
+        .expect("Failed to initialize DateTimeFormatter");
+
+    let date = DateTime::try_new_gregorian_datetime(2020, 10, 14, 13, 21, 28)
+        .expect("Failed to create a datetime.");
+
+    let formatted_date = dtf.format(&date);
+
+    println!("📅: {}", formatted_date);
+}
+```
+
+This has two advantages: it reduces our code size, as `DateTimeFormatter` includes much more functionality than `TypedDateTimeFormatter<Gregorian>`, and it reduces our data size, as static analysis can now determine that we need even fewer keys. The data size improvement could have also been achieved by manually listing the data keys we think we'll need (there's a `--keys` flag), but we risk a runtime error if we're wrong.
+
+This is a common pattern in `ICU4X`, and most of our APIs are designed with data slicing in mind.
+
+Rerunning datagen awards us with a 3KB data blob, which only contains 7 data keys!
```

Comment:
Nit (word choice): the object of the verb "award" is the thing being conferred or bestowed. Consider "reward" instead.